Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -101,9 +104,23 @@ - (dispatch_queue_t)methodQueue
 - (void)imagePickerController:(UIImagePickerController *)picker
 didFinishPickingMediaWithInfo:(NSDictionary<NSString *, id> *)info
 {
-  [self _dismissPicker:picker args:@[
-    [info[UIImagePickerControllerReferenceURL] absoluteString]
-  ]];
+  // Image from PhotoLibrary
+  NSString *imageUri = [info[UIImagePickerControllerReferenceURL] absoluteString];
+  if (imageUri) {
+    [self _dismissPicker:picker args:@[imageUri]];
+
+  } else {
+    // Image from CameraRoll hasn't uri.
+    // We need to save it to the store first.
+    UIImage *originalImage = (UIImage *) [info objectForKey: UIImagePickerControllerOriginalImage];
```

Comment:
nit: can you use [...] syntax for accessing keys? Also, no need to cast here:
    
     UIImage *originalImage = info[UIImagePickerControllerOriginalImage];